+dj/Schema.m

```diff
@@ -43,9 +43,9 @@
         %   manual:   tableName starts with a letter (no prefix)
         %   imported: tableName with a '_'
         %   computed: tableName with '__'
-        allowedTiers = {'lookup' 'manual' 'imported' 'computed' 'job'}
-        tierPrefixes = {'#', '', '_', '__', '~'}
-        tierClasses = {'dj.Lookup' 'dj.Manual' 'dj.Imported' 'dj.Computed' 'dj.Jobs'}
+        allowedTiers = {'lookup' 'manual' 'imported' 'computed' 'job', 'hidden'}
+        tierPrefixes = {'#', '', '_', '__', '~', '$'}
```

Comment:
`~` already denotes hidden tables such as Jobs, External, and Log (in python). We do not need to introduce a new table type/prefix.